skyportal/handlers/api/galaxy.py

```diff
@@ -121,12 +123,98 @@ def get(self, catalog_name=None):
         """
 
         catalog_name = self.get_query_argument("catalog_name", None)
+        localization_dateobs = self.get_query_argument("localizationDateobs", None)
+        localization_name = self.get_query_argument("localizationName", None)
+        localization_cumprob = self.get_query_argument("localizationCumprob", 0.95)
+        includeGeojson = self.get_query_argument("includeGeojson", False)
+
         query = Galaxy.query_records_accessible_by(self.current_user, mode="read")
         if catalog_name is not None:
             query = query.filter(Galaxy.catalog_name == catalog_name)
+
+        if localization_dateobs is not None:
+            if localization_name is not None:
+                localization = (
+                    Localization.query_records_accessible_by(self.current_user)
+                    .filter(Localization.dateobs == localization_dateobs)
```

Comment:
unclear is this is going to work if localization_dateobs is not in the exact string format of dateobs. Would it be safer to coerce this API input with a date parser then recast it to a string of the proper form?

Reply:
It's really fundamentally a string here, and really is about checking if the event exists or not, i.e. the logic about checking that the event exists below. So that seems more the crux of it.